main.go

```diff
@@ -1,13 +1,31 @@
 package main
 
 import (
+	"context"
+	"net/url"
+	"os"
+
 	logger "github.com/kubescape/go-logger"
 	"github.com/kubescape/kubescape/v2/cmd"
+	"github.com/kubescape/kubescape/v2/core/cautils"
+	"go.opentelemetry.io/otel"
 )
 
 func main() {
-	if err := cmd.Execute(); err != nil {
-		logger.L().Fatal(err.Error())
+	ctx := context.Background()
+	// to enable otel, set OTEL_COLLECTOR_SVC=otel-collector:4317
+	if otelHost, present := os.LookupEnv("OTEL_COLLECTOR_SVC"); present {
+		ctx = logger.InitOtel("kubescape",
+			os.Getenv(cautils.BuildNumber),
+			os.Getenv("ACCOUNT_ID"),
```

Comment:
[Required] The initialization of the OTEL should not be here.

Reply:
It's in both locations, but OK I'll remove the CLI part.